app/models/carto/map.rb

```diff
@@ -211,6 +211,9 @@ def ensure_options
     options[:legends] = (legends || true) if options[:legends].nil?
```

Comment:
@gfiorav I actually copied that other line from the one you wrote here :) but agreed I should probably get set up with rubcop...